mzLib/FlashLFQ/ChromatographicPeak.cs

```diff
@@ -39,14 +46,6 @@ public ChromatographicPeak(Identification id, bool isMbrPeak, SpectraFileInfo fi
         /// Expected retention time for MBR acceptor peaks (mean)
         /// </summary>
         public double? RtHypothesis { get; private set; }
-        /// <summary>
-        /// Std. Dev of retention time differences between MBR acceptor file and donor file, used if # calibration points < 6
-        /// </summary>
-        public double? RtStdDev { get; private set;  }
```

Comment:
i realize you are not using rthypothesis. but do we not need rtstdev or interquartile? maybe not.